plugins/janus_videoroom.c

```diff
@@ -2582,6 +2586,11 @@ static janus_videoroom_subscriber_stream *janus_videoroom_subscriber_stream_add(
 	stream->type = ps->type;
 	stream->acodec = ps->acodec;
 	stream->vcodec = ps->vcodec;
+	if (stream->vcodec == JANUS_VIDEOCODEC_H264) {
+        stream->h264_profile = g_strdup(ps->fmtp);
+	} else if (stream->vcodec == JANUS_VIDEOCODEC_VP9) {
+	    stream->vp9_profile = g_strdup(ps->fmtp);
+	}
```

Comment:
There's no check to see if `ps->fmtp` exists: if it's `NULL`, I think `g_strdup` will segfault.

Please fix the indentation (you're using spaces, code style requires tabs). Besides, per code style there should be no space between `if` and the bracket (`if (` --> `if(`).

Reply:
👍